package/Dockerfile

```diff
@@ -0,0 +1,5 @@
+ARG UBI_IMAGE=registry.access.redhat.com/ubi8/ubi-minimal:latest
+FROM ${UBI_IMAGE}
```

Comment:
need to run the following to ensure packages are up to date:
```
RUN microdnf update -y && \ 
    rm -rf /var/cache/yum
```

Reply:
Done.